core-web/libs/portlets/dot-ema/src/lib/shared/models.ts

```diff
@@ -38,3 +38,12 @@ export interface ContainerActionPayload {
     contentletID: string;
     personaTag?: string;
 }
+
+export interface NavigationBarItem {
+    icon?: string;
+    iconURL?: string;
+    iconURLActive?: string;
```

Comment:
Why don't we just use a svg file? this seems very complex to me.

Reply:
I agree